addons/payment_authorize/models/authorize_request.py

```diff
@@ -58,9 +58,17 @@ def _make_request(self, operation, data=None):
 
         messages = response.get('messages')
         if messages and messages.get('resultCode') == 'Error':
+            err_msg = messages.get('message')[0].get('text', '')
```

Comment:
Can it happen that the error message is missing from the response?

Reply:
I added it for safety because we concatenate later and `None + "\n"` would blow up. I just checked [the XSD](https://api.authorize.net/xml/v1/schema/AnetApiSchema.xsd) and `text` is required in fact. So I can remove the `''` default if you prefer that. Being explicit like this make it clear it's safe without having to dive into the XSD, but I don't have a strong opinion on it :).

Here's the relevant XSD snippet:

```xml
<xs:complexType name="messagesType">
  <xs:sequence>
    <xs:element name="resultCode" type="anet:messageTypeEnum"/>
    <xs:element name="message" maxOccurs="unbounded">
      <xs:complexType>
        <xs:sequence>
          <xs:element name="code" type="xs:string"/>
          <xs:element name="text" type="xs:string"/>
        </xs:sequence>
      </xs:complexType>
    </xs:element>
  </xs:sequence>
</xs:complexType>
```

Reply:
Nice, then the preferred way to do it is `err_msg = messages.get('message')[0]['text']`. This way, any dev immediately knows that they can assume `err_msg` is a string and not `None` without double-checking the XSD.

Reply:
It feels strange to change a defensive `get()` to an aggressive `[]` in a stable release, but I understand your point. I triple-checked and it should be safe, so I made the change.

Reply:
Ok, let's not change the `get` then, I changed it back. It will help me sleep better at night 😄.